test/conformance/service_test.go

```diff
@@ -351,7 +388,4 @@ func TestReleaseService(t *testing.T) {
 		[]string{expectedThirdRev, expectedSecondRev, expectedFirstRev})
 }
 
-// TODO(jonjohnsonjr): LatestService roads less traveled.
-// TODO(jonjohnsonjr): PinnedService happy path.
-// TODO(jonjohnsonjr): PinnedService roads less traveled.
```

Comment:
Intended to remove these, or bad merge?

Reply:
Intended to remove as they are no longer relevant. I probably should have separated into another PR, but figured it was just comments so there is no chance of having to revert. PinnedService is deprecated and LatestService now has a decent amount of coverage.